web/src/adapters/authentication.js

```diff
@@ -32,5 +32,41 @@ async function registerNewUser({ firstname, lastname, email, password, birthday
   }
 }
 
-export { registerNewUser };
+async function loginUser({ email, password }) {
+  try {
+    const response = await fetch(`${AUTHENTICATION_URL}login`, {
```

Comment:
The endpoint is authenticate 👎
```suggestion
    const response = await fetch(`${AUTHENTICATION_URL}authenticate`, {
```